debian/test-launch-system76-keyboard-configurator.install

```diff
@@ -1 +1 @@
-usr/share/applications/com.system76.keyboardconfiguratorlaunchtest.desktop
+linux/com.system76.keyboardconfiguratorlaunchtest.desktop usr/share/applications/com.system76.keyboardconfiguratorlaunchtest.desktop
```

Comment:
The part on the right here is installation directory, so this will result in a `com.system76.keyboardconfiguratorlaunchtest.desktop` directory containing a file of the same name.

(Yes, Debian packaging is an arcane dark magic.)

Reply:
Ah I got a little too tab complete happy. Fixed. Thanks

Reply:
I don't know how many times I've gotten `.install` files wrong the first time I tried to write them. :laughing: